### DISCUSSION TITLE
Changes are not being deployed

### DISCUSSION THREAD

**Discussion by VivekJadeja** [community/community#22825] [Other Feature Feedback, Questions, & Ideas]
<p>Hello all,</p>
<p>I have been trying to update my website that is hosted on GitHub Pages, but when I pushed my update a few days ago, there was no change on the site.</p>
<p>I can clearly see that the change has been pushed to my repository, but nothing has changed on the actual website. The changes, by the way, were to the title of the page in public-&gt;index.html, and the modification of a link in src-&gt;components-&gt;carousel.js.</p>
<p>If anyone can help me with this issue, I’d really appreciate it!</p>
<p>-Vivek</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/vivekjadeja">@VivekJadeja</a>, welcome to the GitHub Support Communtiy!</p>
<p>Can you drop a link to your repository so we can take a look at what’s going on?</p>

**Comment by VivekJadeja** [comment]
<p>Yes! Here is the link to the repository in which I am having this issue: <a href="https://github.com/VivekJadeja/VivekJadeja.github.io" rel="noopener nofollow ugc">https://github.com/VivekJadeja/VivekJadeja.github.io</a></p>
<p>This is for the website that I am hosting on GitHub Pages, as stated above.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Thanks for posting the link! It looks like you’ve already managed to figure it out but just to leave the answer here for any others that end up in a similar situation:</p>
<p>It looks like your project uses a build and deploy step that’s defined in your <code>package.json</code> file.</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/VivekJadeja/VivekJadeja.github.io/blob/0c064ef41a8e7d91668ec5d4efff01ea743b02fb/package.json#L24" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/VivekJadeja/VivekJadeja.github.io/blob/0c064ef41a8e7d91668ec5d4efff01ea743b02fb/package.json#L24" target="_blank" rel="noopener">VivekJadeja/VivekJadeja.github.io/blob/0c064ef41a8e7d91668ec5d4efff01ea743b02fb/package.json#L24</a></h4>
<pre class="onebox"><code class="lang-json"><ol class="start lines" start="14" style="counter-reset: li-counter 13 ;">
<li>  "react-bootstrap": "^1.3.0",</li>
<li>  "react-dom": "^16.13.1",</li>
<li>  "react-router-bootstrap": "^0.25.0",</li>
<li>  "react-router-dom": "^5.2.0",</li>
<li>  "react-scripts": "3.4.3",</li>
<li>  "react-spring": "^8.0.27"</li>
<li>},</li>
<li>"scripts": {</li>
<li>  "start": "react-scripts start",</li>
<li>  "predeploy": "npm run build",</li>
<li class="selected">  "deploy": "gh-pages -d build",</li>
<li>  "build": "react-scripts build",</li>
<li>  "test": "react-scripts test",</li>
<li>  "eject": "react-scripts eject"</li>
<li>},</li>
<li>"eslintConfig": {</li>
<li>  "extends": "react-app"</li>
<li>},</li>
<li>"browserslist": {</li>
<li>  "production": [</li>
<li>    "&gt;0.2%",</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I believe that the issue you were seeing was because you pushed the changes you made to the <code>master</code> branch of your repository but didn’t run this deploy step after, meaning the built version of your site wasn’t pushed to your <code>gh-pages</code> branch that serves your site’s code. To make sure this doesn’t happen again in future you’ll need to run <code>npm run deploy</code> after pushing the latest changes to your <code>master</code> branch.</p>
